package-common.sh

```diff
@@ -1,7 +1,17 @@
 # !/bin/bash
```

Comment:
Could we move all packaging / publishing files to own folder?

Reply:
I can try to make it better next time we do release. Haven't thought that it yet matters and haven't had good vision about it. But at least some of those which are currently in root may fit to scripts folder... 🤔